src/renderer/components/screens/AccountSetupScreen.tsx

```diff
@@ -61,6 +61,21 @@ export default function AccountSetupScreen({
     }
   }
 
+  const onKeyDown = (ev: KeyboardEvent) => {
```

Comment:
eslint's rules-of-react hooks check complains here, run `npm run test` or look bellow on gh.
You probably want a useCallback or useMemo here.